extensions/roc-plugin-style-css/src/config/roc.config.meta.js

```diff
@@ -24,6 +24,10 @@ export default {
                     description: 'If CSS source maps should be enabled.',
                     validator: isBoolean,
                 },
+                minimize: {
+                    description: 'Activate or Desactivate css minimization',
```

Comment:
English spelling is `deactivate`

Reply:
I think the description should reflect how it works a bit better matching how others are worded; `If minification should be enabled.`.